config/locales/en.yml

```
@@ -820,12 +820,21 @@ en:
       back: "Back"
       #submit:
     can_only_create_5: 'You can only add 5 labels'
+    too_long: 'The entered label name is too long'
+    cannot_be_empty: 'The entered label name is empty'
 
     edit:
       header: "Editing Label"
     new:
       header: "New Label"
       selection_suffix: " (new)"
+    index:
+      merge: 'Merge And Rename To'
+      delete: 'Delete selected'
+      change_color_to: 'Change Color To'
```

I would go with [Capitalization](https://capitalizemytitle.com/), similar to a paper here:

```suggestion
      merge: 'Merge and Rename to'
      delete: 'Delete selected'
      change_color_to: 'Change Color to'
```